src/apps/chatter/hooks/use-room-password.js

```diff
@@ -36,6 +38,9 @@ export function useRoomPassword (roomId, password) {
         console.log('RECEIVED INIT RESPONSE COMMAND: ', parsedMessage)
         setIsChecking(false)
         setIsCorrectPassword(parsedMessage.authorized)
+        if (parsedMessage.authorized) {
+          dispatch({ type: 'SET_ROOM_PASSWORD', roomId, password })
```

Comment:
Better to use action creators: https://github.com/inputlogic/elements/blob/master/components/use-request/reducer.js#L15

https://redux.js.org/basics/actions/#action-creators